rest/model/data/meta.go

```diff
@@ -180,10 +183,7 @@ func FormatInterface(i interface{}) string {
 		}
 		return strings.Join(slc, ",")
 	case FeedPtr:
-		data, err := json.Marshal(v)
-		if err != nil {
-			panic(err)
-		}
+		data, _ := json.Marshal(v)
```

Comment:
Curious, why aren't we handling the error anymore?  I agree with removing the panic, but perhaps we could bubble up the error and return it instead?

Reply:
Spoke IRL